docs/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "docs",
   "private": true,
+  "type": "module",
```

Comment:
This resolves a build issue and is an important change:
<img width="1191" alt="CleanShot 2023-09-20 at 14 47 54@2x" src="https://github.com/axflow/axflow/assets/1666947/440820ce-b689-4304-8366-8d4ce39bb027">